cdap-app-fabric/src/main/java/io/cdap/cdap/internal/credential/store/Utils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.internal.credential.store;
+
+import io.cdap.cdap.proto.id.CredentialProfileId;
+import java.util.regex.Pattern;
+
+/**
+ * Utility methods for credential provisioning stores.
+ */
+public class Utils {
```

Comment:
nit: utility classes should have a more descriptive name like CredentialProfiles to try and keep them more focused.

Reply:
Done